zio-http/shared/src/main/scala/zio/http/codec/HttpCodec.scala

```diff
@@ -334,7 +334,7 @@ object HttpCodec extends ContentCodecs with HeaderCodecs with MethodCodecs with
   def unused: HttpCodec[Any, ZNothing] = Halt
 
   final case class Enumeration[Value](unit: Unit) extends AnyVal {
-    def apply[AtomTypes, Sub1 <: Value: ClassTag, Sub2 <: Value: ClassTag](
+    def f2[AtomTypes, Sub1 <: Value: ClassTag, Sub2 <: Value: ClassTag](
```

Comment:
Why were these all renamed??

Reply:
After changing the order, the compiler could not infer the right apply. So I gave them explicit names.